src/app/pages/components/home/design-list.tsx

```diff
@@ -71,12 +71,11 @@ export const DesignList: FunctionComponent<DesignListProps> = (
 
     const actionsFor = (design: any): IAction[] => {
         return [
-            { title: "View details", onClick: () => setSelectedDesign(design) },
-            { isSeparator: true, },
+            { title: "Details", onClick: () => setSelectedDesign(design) },
```

Comment:
Can you restore the first separator?  I think we should have:

* Details
* Separator
* Other Actions
* Separator
* Delete

Reply:
![Screenshot 2022-06-22 at 16 45 38](https://user-images.githubusercontent.com/11743717/175074999-4a24f668-dba7-4bb7-b6e4-f991e9154e1e.png)

I was keeping it consistent with Kafka.

Reply:
I see.  No separators at all in Kafka, eh?  Not even for Delete!

Reply:
![Screenshot 2022-06-22 at 17 12 58](https://user-images.githubusercontent.com/11743717/175081472-3d89418e-8d17-4c12-83ae-f4b4a49273f9.png)

Yeah, same in Service Registry, no separator for Delete!

Reply:
I don't know. :(

Reply:
My vote is to remove the separator for consistency for now. If we want it changed it should be applied across the board and we don't know if there was a deliberate decision to do it this way.

Reply:
Works for me.

Reply:
OK thanks Sim.  Will do our best.  Not sure if we can do the group titles yet - I don't see an obvious way with the table control but I haven't dug into it seriously yet.